test/e2e/page-objects/pages/send/send-token-page.ts

```diff
@@ -147,14 +197,40 @@ class SendTokenPage {
     await this.driver.pasteIntoField(this.inputRecipient, recipientAddress);
   }
 
+  async fillHexInput(hex: string): Promise<void> {
+    console.log(`Filling hex input with: ${hex}`);
+    await this.driver.pasteIntoField(this.hexInput, hex);
+  }
+
+  async getHexInputValue(): Promise<string> {
+    console.log('Getting value from hex input');
+    const hexInputElement = await this.driver.waitForSelector(this.hexInput);
```

Comment:
add `await this.driver.waitForNonEmptyElement(hexInputElement); before get value to mitigate flakiness